docs/HoloLens/VideoRecording.md

```diff
@@ -0,0 +1,36 @@
+---
+id: VideoRecording
+title: Video Recording
+---
+
+Augmented Reality is inherently social, so it's common to want to share your time in an experience with others. 
+
+In any experience, it's possible to start a recording directly in Enklu that gets emailed to you directly. 
+No more fussing with cables to export videos off your device anymore.
+
+![Video Recording](/img/product/hololens/video-recording.gif)
+
+### Starting a Recording
+
+1. Open the experience menu by saying "menu".
+2. Select "Record" with your gaze, and follow the prompt.
+3. While waiting for the countdown, make sure you're positioning is good.
+
+Video recording can also be triggered faster by saying "record".
+
+While recording is active, the HoloLens will display a recording icon in the upper left corner of the screen. This is not visible in your recordings.
+
+> Recordings capture the visible area where Holograms can be seen. The area outside of the device's Field of View will not be cropped from the final video.
+
+### Stopping a Recording
+
+After you've recorded everything you want, say "stop". You will be notified that the recording is being uploaded.
+
+Check your email, where your video will be sent to you shortly.
+
+### Making the best recording
+
+When making a recording, be mindful that the viewer isn't in control of the device. 
+Sharp and sudden movements will appear disorienting when viewing the video.
```

Comment:
When making a recording, be mindful that sharp and sudden movements may be disorienting for the user viewing the video.